aptos-move/move-examples/post_mint_reveal_nft/sources/whitelist.move

```diff
@@ -0,0 +1,102 @@
+module post_mint_reveal_nft::whitelist {
```

Comment:
Do any functions here need to be public(friend) at all?

Reply:
not really, I am aware that we already have a signer for writing functions, and the reading functions don't have to be only readable from the `minting` module - but wondering if it's better to have better access control since rn we are only using this module in `minting.move`

Reply:
actually yes, looking thru this again - we are calling `deduct_user_minting_amount` to deduct the remaining minting amount from the whitelist address map, but we are not taking an admin signer to call this function (since this function is being called in `mint_source_certificate`, which only takes in the `minter`'s signer)